### ISSUE TITLE
dvdauthor FIXUPHACK gone

### ISSUE THREAD

**Issue by peabee** [puppylinux-woof-CE/woof-CE#1289]
packages-templates/dvdauthor_FIXUPHACK
has disappeared - this hack is still needed:
#120820 ubuntu precise: spumux needs pkg imagemagick, too big...
[ -f usr/bin/spumux ] && rm -f usr/bin/spumux

**Comment by wdlkmpx** [comment]
As far as i know only tahrpup is affected, but it uses a pet pkg.
https://packages.ubuntu.com/trusty/dvdauthor

I think xenialpup and upupbb use dvdauthor from ubuntu

https://packages.ubuntu.com/xenial/dvdauthor
https://packages.ubuntu.com/bionic/dvdauthor
https://packages.ubuntu.com/cosmic/dvdauthor

Bonus:
https://packages.debian.org/stretch/dvdauthor

dvdauthor is a salix pkg (slackware), but i don't see imagemagick as a dependency
Dependencies:
bzip2 expat fontconfig freetype fribidi glib2 harfbuzz libdvdread libpng libxml2 xz zlib 

So basically, if it needs imagemagick, it's a broken package. i guess that's why it became a pet pkg.. unless i'm wrong

**Comment by wdlkmpx** [comment]
upupcc
yes|dvdauthor|dvdauthor|exe,dev,doc,nls|

File: /usr/bin/spumux

Missing dependencies:


Extended information:
	linux-gate.so.1 (0xb777c000)
	libxml2.so.2 => /usr/lib/libxml2.so.2 (0xb75ab000)
	libpng16.so.16 => /usr/lib/libpng16.so.16 (0xb7571000)
	libm.so.6 => /lib/libm.so.6 (0xb746b000)
	libfreetype.so.6 => /usr/lib/libfreetype.so.6 (0xb73ae000)
	libfribidi.so.0 => /usr/lib/libfribidi.so.0 (0xb7392000)
	libfontconfig.so.1 => /usr/lib/libfontconfig.so.1 (0xb7347000)
	libc.so.6 => /lib/libc.so.6 (0xb7166000)
	libdl.so.2 => /lib/libdl.so.2 (0xb7160000)
	libicuuc.so.60 => /usr/lib/libicuuc.so.60 (0xb6fa1000)
	libz.so.1 => /lib/libz.so.1 (0xb6f82000)
	liblzma.so.5 => /lib/liblzma.so.5 (0xb6f56000)
	/lib/ld-linux.so.2 (0xb777d000)
	libexpat.so.1 => /lib/libexpat.so.1 (0xb6f19000)
	libuuid.so.1 => /lib/libuuid.so.1 (0xb6f0f000)
	libpthread.so.0 => /lib/libpthread.so.0 (0xb6eed000)
	libicudata.so.60 => /usr/lib/libicudata.so.60 (0xb5543000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0xb53bc000)
	libgcc_s.so.1 => /lib/libgcc_s.so.1 (0xb539e000)

	Version information:
	/usr/bin/spumux:
		libm.so.6 (GLIBC_2.0) => /lib/libm.so.6
		libxml2.so.2 (LIBXML2_2.6.0) => /usr/lib/libxml2.so.2
		libxml2.so.2 (LIBXML2_2.5.0) => /usr/lib/libxml2.so.2
		libxml2.so.2 (LIBXML2_2.4.30) => /usr/lib/libxml2.so.2
		libpng16.so.16 (PNG16_0) => /usr/lib/libpng16.so.16
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.2) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
	/usr/lib/libxml2.so.2:
		libz.so.1 (ZLIB_1.2.3.3) => /lib/libz.so.1
		libz.so.1 (ZLIB_1.2.2.3) => /lib/libz.so.1
		liblzma.so.5 (XZ_5.0) => /lib/liblzma.so.5
		libdl.so.2 (GLIBC_2.1) => /lib/libdl.so.2
		libdl.so.2 (GLIBC_2.0) => /lib/libdl.so.2
		libm.so.6 (GLIBC_2.0) => /lib/libm.so.6
		libc.so.6 (GLIBC_2.15) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.28) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.2) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.7) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.2) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
	/usr/lib/libpng16.so.16:
		libm.so.6 (GLIBC_2.0) => /lib/libm.so.6
		libz.so.1 (ZLIB_1.2.9) => /lib/libz.so.1
		libz.so.1 (ZLIB_1.2.3.4) => /lib/libz.so.1
		libc.so.6 (GLIBC_2.11) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
	/lib/libm.so.6:
		ld-linux.so.2 (GLIBC_PRIVATE) => /lib/ld-linux.so.2
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libc.so.6 (GLIBC_PRIVATE) => /lib/libc.so.6
	/usr/lib/libfreetype.so.6:
		libc.so.6 (GLIBC_2.11) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libpng16.so.16 (PNG16_0) => /usr/lib/libpng16.so.16
	/usr/lib/libfribidi.so.0:
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
	/usr/lib/libfontconfig.so.1:
		libc.so.6 (GLIBC_2.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.2) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.7) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libuuid.so.1 (UUID_1.0) => /lib/libuuid.so.1
		libpthread.so.0 (GLIBC_2.2) => /lib/libpthread.so.0
		libpthread.so.0 (GLIBC_2.0) => /lib/libpthread.so.0
	/lib/libc.so.6:
		ld-linux.so.2 (GLIBC_2.3) => /lib/ld-linux.so.2
		ld-linux.so.2 (GLIBC_PRIVATE) => /lib/ld-linux.so.2
		ld-linux.so.2 (GLIBC_2.1) => /lib/ld-linux.so.2
	/lib/libdl.so.2:
		ld-linux.so.2 (GLIBC_PRIVATE) => /lib/ld-linux.so.2
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_PRIVATE) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
	/usr/lib/libicuuc.so.60:
		libgcc_s.so.1 (GCC_3.0) => /lib/libgcc_s.so.1
		libstdc++.so.6 (CXXABI_1.3.8) => /usr/lib/libstdc++.so.6
		libstdc++.so.6 (GLIBCXX_3.4) => /usr/lib/libstdc++.so.6
		libstdc++.so.6 (CXXABI_1.3) => /usr/lib/libstdc++.so.6
		libdl.so.2 (GLIBC_2.0) => /lib/libdl.so.2
		libdl.so.2 (GLIBC_2.1) => /lib/libdl.so.2
		libm.so.6 (GLIBC_2.0) => /lib/libm.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.7) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libpthread.so.0 (GLIBC_2.0) => /lib/libpthread.so.0
		libpthread.so.0 (GLIBC_2.3.2) => /lib/libpthread.so.0
	/lib/libz.so.1:
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
	/lib/liblzma.so.5:
		libdl.so.2 (GLIBC_2.1) => /lib/libdl.so.2
		libdl.so.2 (GLIBC_2.0) => /lib/libdl.so.2
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.17) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libpthread.so.0 (GLIBC_2.1) => /lib/libpthread.so.0
		libpthread.so.0 (GLIBC_2.3.3) => /lib/libpthread.so.0
		libpthread.so.0 (GLIBC_2.3.2) => /lib/libpthread.so.0
		libpthread.so.0 (GLIBC_2.0) => /lib/libpthread.so.0
	/lib/libexpat.so.1:
		libc.so.6 (GLIBC_2.25) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
	/lib/libuuid.so.1:
		ld-linux.so.2 (GLIBC_2.3) => /lib/ld-linux.so.2
		libc.so.6 (GLIBC_2.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.25) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.28) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
	/lib/libpthread.so.0:
		ld-linux.so.2 (GLIBC_2.1) => /lib/ld-linux.so.2
		ld-linux.so.2 (GLIBC_PRIVATE) => /lib/ld-linux.so.2
		libc.so.6 (GLIBC_2.3.2) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.2) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libc.so.6 (GLIBC_PRIVATE) => /lib/libc.so.6
	/usr/lib/libstdc++.so.6:
		libm.so.6 (GLIBC_2.0) => /lib/libm.so.6
		ld-linux.so.2 (GLIBC_2.3) => /lib/ld-linux.so.2
		libgcc_s.so.1 (GCC_4.2.0) => /lib/libgcc_s.so.1
		libgcc_s.so.1 (GLIBC_2.0) => /lib/libgcc_s.so.1
		libgcc_s.so.1 (GCC_3.3) => /lib/libgcc_s.so.1
		libgcc_s.so.1 (GCC_3.0) => /lib/libgcc_s.so.1
		libc.so.6 (GLIBC_2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.18) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.16) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.17) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.3.2) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.2) => /lib/libc.so.6
	/lib/libgcc_s.so.1:
		libc.so.6 (GLIBC_2.2.4) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.1.3) => /lib/libc.so.6
		libc.so.6 (GLIBC_2.0) => /lib/libc.so.6

**Comment by peabee** [comment]
LxPupSc-18.12 is moving to woof-ce testing branch....
uses:
http://ponce.cc/slackware/slackware-current/packages/dvdauthor-0.7.2-i486-1ponce.txz
and this has the dependency....

upupcc uses dvdauthor_0.7.0-2build1_i386.deb

I can fix LxPupSc in _00build.conf or use different version

Preview: https://sourceforge.net/projects/lxpup/files/Other/test/LxPupSc-18.12T-k64-uefi.iso/download
